lib/rspec/core/formatters/documentation_formatter.rb

```diff
@@ -22,7 +22,10 @@ def example_group_started(notification)
         end
 
         def example_group_finished(_notification)
-          @group_level -= 1
+          if @group_level > 0
```

Comment:
Our preference here is to either have inline statements e.g. `@group_level -= 1 if @group_level > 0` but stylistically here I think `return @group_level if @group_level == 0` might be nicer. As it stands however our rubocop is picking this up as a fail :(